public/locales/en-GB/app.json

```
@@ -81,8 +81,8 @@
     "call_ended_heading": "Call ended",
     "failed_heading": "Failed to join",
     "failed_text": "Call not found or is not accessible.",
-    "knock_reject_body": "The room members declined your request to join.",
-    "knock_reject_heading": "Not allowed to join",
+    "knock_reject_body": "Your request was declined.",
+    "knock_reject_heading": "Access Denied",
```

```suggestion
    "knock_reject_heading": "Access denied",
```

(we don't appear to be using title case in general)